Dockerfile

```
@@ -0,0 +1,20 @@
+FROM rhel8/go-toolset:1.12.8 AS builder
+WORKDIR /opt/app-root/src/go/src/github.com/knative/client
+COPY . .
+RUN TAG="v0.8.0" make build
+
+FROM ubi8:8-released
+COPY --from=builder /opt/app-root/src/go/src/github.com/knative/client/kn /usr/bin/kn
+
+LABEL \
+      com.redhat.component="openshift-serverless-1-tech-preview-client-kn-rhel8-container" \
+      name="openshift-serverless-1-tech-preview/client-kn-rhel8" \
+      version="0.8.0" \
+      summary="Red Hat OpenShift Serverless 1 Client kn container" \
```

```suggestion
      summary="Red Hat OpenShift Serverless 1 - kn CLI" \
```
Shouldn't we add also 'cli' to the summary and/or description ?